src/app/components/common/BorderScreen/BorderScreen.tsx

```diff
@@ -81,6 +87,7 @@ const BorderScreen = (props: Props) => {
                 </Grid>
               </Grid>
             )}
+            {sideElement && sideElement}
```

Comment:
What does this statement do?

Reply:
If we provide some side components, for example counter